README.md

```diff
@@ -13,6 +13,20 @@ There are some cargo features that are highly relevant for developers:
 * `skip-ias-check`: allow registering enclaves without attestation report.
 * `skip-extrinsic-filtering`: We have a defensive filter for transfer extrinsics as we have an old solo-node running for archive purposes, which mustn't allow transfers. The filter can be deactivated with this feature.
 
+## Versioning
+There are two important version parameters in the runtime that change behaviour.
+* `spec_version` always needs to be updated when the runtime logic changes.
+* `transaction_version` always needs to updated when:
+  * extrinsics are removed
+  * extrinsics are changed
+  * extrinsics are added somewhere in between
+Extrinsics being added to the end are fine as they don't interfere with the existing call enum that the runtime macro
```

Comment:
In my understanding if I a person adds to the end of the extrinsics, then they:
- will not have to bump `transaction_version`
- will not have to bump `spec_version` (as a new extrinsics addition is considered a runtime logic change?)

Is that right?

------
[Also the substrate docs says that 
](https://docs.substrate.io/build/upgrade-the-runtime/)
> If this [ `transaction_version`] number is updated, then the `spec_version` must also be updated.

I think this is implied here as well, nevertheless could you please state it explicitly as well?

Reply:
Good point! I have updated the readme to just link to the official docs to not create any ambiguities. :)